bee-ledger/src/workers/error.rs

```diff
@@ -7,76 +7,57 @@ use bee_message::{address::Address, milestone::MilestoneIndex, Error as MessageE
 
 use crate::{
     types::{Balance, Error as TypesError, Unspent},
-    workers::snapshot::error::Error as SnapshotError,
+    workers::{pruning::error::PruningError, snapshot::error::Error as SnapshotError},
 };
 
 /// Errors occurring during ledger workers operations.
 #[derive(Debug, thiserror::Error)]
+#[allow(missing_docs)]
 pub enum Error {
-    /// Snapshot error.
```

Comment:
Still not rebased ?

Reply:
what the heck?..it was 3 days ago ... see the forced push.

Reply:
I don't know what's going. Maybe I have accidentally added them back during the rebase :shrug:

Reply:
It's in stardust that I removed them. Sorry, ignore me.